DESCRIPTION

```diff
@@ -8,7 +8,7 @@ Authors@R: c(
     person("Hadley", "Wickham", , "hadley@rstudio.com", "aut"),
     person("Davis", "Vaughan", , "davis@rstudio.com", "aut"),
     person("RStudio", role = "cph"))
-License: GPL-2
+License: MIT
```

Comment:
Should be `+ file LICENSE` I think?  If you're not already using it, `use_mit_license("RStudio")` should be all you need

Reply:
Done and done (>ლ)